library/certificate_request.py

```diff
@@ -359,7 +358,7 @@ def _get_argument_spec():
             directory=dict(type="str", default="/etc/pki/tls"),
             provider_config_directory=dict(type="str", default="/etc/certmonger"),
             provider=dict(type="str", default="certmonger"),
-            key_size=dict(type="int", default=2048),
+            key_size=dict(type="int", default=None),
```

Comment:
so the certmonger provider will by default issue 2048 key_size?  On el7, el8, el9, and fedora?

Reply:
Yes, by default it will be 2048.

Also, there's a warning in the role documentation that if the user wants to ensure a specific key size, the parameter `key_size` should be explicitly used as the default value could be changed depending on the role version.